client/src/app/shared/models/poll/poll.ts

```diff
@@ -35,6 +35,8 @@ export class Poll extends BaseDecimalModel<Poll> {
     public global_yes: boolean;
     public global_no: boolean;
     public global_abstain: boolean;
+    public is_pseudoanonymized: boolean;
+    public entitled_users_at_stop: EntitledUsersEntry[];
```

Comment:
@jsangmeister @FinnStutzenstein I suppose this is not the correct place for `entitled_users_at_stop` in OS4 anymore? Where should it go?

Reply:
I think, this is the right place.